### ISSUE TITLE
Add proper README.md before submission

### ISSUE THREAD

**Issue by jerichosy** [jerichosy/CSOPESY-MP#3]
> A brief documentation to compile and run the program should also be included.

Check what needs to be added (e.g., specs, manual)
- [x] Add minimum version requirement for Python of 3.10 (for use of [`bisect` _key_ parameter](https://docs.python.org/3/library/bisect.html#:~:text=the%20right%20slice.-,Changed%20in%20version%203.10%3A%20Added%20the%20key%20parameter.,-bisect.insort_left))
- [x] Add speedups advice like using PyPy
- [x] Add instructions in running the program
- [x] (Optional) Add instructions in running the test script

~~Rename existing [`README.md`](https://github.com/jerichosy/CSOPESY-MP/blob/576f1ae4390395022f32334741959280b3204961/README.md) with notes and stuff to something like `NOTES.md`. **Do this only after the notes is no longer needed to be the default** (i.e., when the project is done).~~
